lib/ember-build.js

```diff
@@ -59,7 +66,11 @@ EmberBuild.prototype._concatenateES6Modules = concatenateES6Modules;
 EmberBuild.prototype._buildTemplateCompilerTree = buildTemplateCompilerTree;
 
 EmberBuild.prototype._init = function init() {
-  this._packages = null;
+  this._packages      = null;
+  this._name          = null;
+  this._namespace     = null;
+  this._skipRuntime   = null;
+  this._skipTemplates = null;
```

Comment:
@seanpdoyle actually, good point. It seems like we don't need to dynamically include/exclude runtime or template compiler (happens once per build). So I think it's fine for it to be a environment flag.